templates/account/payment-methods/index.html

```diff
@@ -28,53 +26,82 @@
+            You have one or more pending payments. Use your saved card to <a id="try-again-button">retry payment</a>. <i id="try-again-spinner"
+    class="p-icon--spinner u-animation--spin is-dark u-hide"></i>
+          </p>
         </div>
       </div>
 
-      <h1>Payment methods</h1>
+      <h1>Payment method</h1>
 
-      <div id="default-payment-method-section" class="row {% if not default_payment_method["id"] %}u-hide{% endif %}" style="margin-top: 2rem;">
+      <div id="default-payment-method-section"
+           class="row {% if not default_payment_method.id %}u-hide{% endif %}"
+           style="margin-top: 2rem">
         <div class="col-4 current-payment-method">
           {% with brand=default_payment_method["brand"] %}
             {% include "account/payment-methods/_card-logo.html" %}
           {% endwith %}
-          <p><span style="text-transform: capitalize;">{{default_payment_method["brand"]}}</span> ending in {{default_payment_method["last4"]}}</p>
-          </div>
+          <p>
+            <span style="text-transform: capitalize;">{{ default_payment_method["brand"] }}</span> ending in {{ default_payment_method["last4"] }}
+          </p>
+        </div>
         <p class="col-4 u-text--muted">Default payment method</p>
         <div class="col-4 u-align--right">
           <button class="p-button" id="edit-payment-details">Edit</button>
+          <button class="p-button--base"
+                  id="remove-payment-details"
+                  aria-controls="remove-payment-modal">Remove</button>
         </div>
       </div>
 
-      {% if default_payment_method %}
-        <div id="edit-payment-method-section" class="row {% if default_payment_method["id"] %}u-hide{% endif %}" style="margin-top: 2rem;">
-          <div class="col-8">
-              <div id="card-element"></div>
-              <span id="card-errors" class="p-form-validation__message u-hide"></span>
-          </div>
-          <div class="col-4 u-align--right">
-            {% if default_payment_method["id"] %}<button class="p-button" id="cancel-payment-details">Cancel</button>{% endif %}
-            <button class="p-button--positive" id="update-payment-details" disabled>Update</button>
+      <div id="edit-payment-method-section"
+           class="row u-hide"
+           style="margin-top: 2rem">
+        <div class="col-8">
+          <div id="card-element"></div>
+          <span id="card-errors" class="p-form-validation__message u-hide"></span>
+        </div>
+        <div class="col-4 u-align--right">
+          <button class="p-button" id="cancel-payment-details">Cancel</button>
+          <button class="p-button--positive" id="update-payment-details" disabled>Update</button>
+        </div>
+        <div class="col-8"></div>
+      </div>
+      <hr class="{% if not default_payment_method.id %}u-hide{% endif %}" />
```

Comment:
I think the line should always show?

Reply:
This is the line at the bottom of the card form. I think it should not be there when there's no card?

Reply:
sorry, I meant the line should display on editing :)